Stylesheets/Components/alert.css

```diff
@@ -0,0 +1,33 @@
+body {
+    background-color: var(--color-background-primary);
+}
+
+.alert-box {
```

Comment:
Since I cannot see your HTML, I assume you are using this class as container for all alerts. 

If it is not the case i.e., you are using `.alert-box` just as a wrapper to each alert box individually, then flex-gap property does not make sense.

Reply:
Okay, I'll do the rectifications. Thank you for the review